app/routes.py

```diff
@@ -1,2 +1,88 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response, request
+from .models.planet import Planet
+from app import db
 
+def error_message(message, status_code):
```

Comment:
you could move your helper functions to a file `helper_functions.py` to clean up your routes file